lib/BankAccount.php

```diff
@@ -14,7 +14,6 @@
  * @property string $country
  * @property string $currency
  * @property string $customer
- * @property bool $default_for_currency
```

Comment:
This one should stay. The class is shared between bank accounts on Customers and bank accounts on Accounts and that property exists for the latter.